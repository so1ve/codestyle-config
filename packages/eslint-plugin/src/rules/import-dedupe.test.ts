import { run } from "./_test";
import rule, { RULE_NAME } from "./import-dedupe";

const valid = ['import { a } from "foo";'];
const invalid = ['import { a, b, a, a, c, a , } from "foo";'];

run({
	name: RULE_NAME,
	rule,
	valid,
	invalid,
});
