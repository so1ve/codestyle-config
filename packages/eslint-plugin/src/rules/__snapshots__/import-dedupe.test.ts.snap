// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`import-dedupe > invalid > Invalid #0: import { a, b, a, a, c, a , } from "foo"; 1`] = `"import { a, b,   c,  } from "foo";"`;
